### ISSUE TITLE
uconv -V command fails if uconv is not installed

### ISSUE THREAD

**Issue by hattmarris** [smashedtoatoms/asdf-postgres#64]
**Describe the bug**

`uconv -V` fails if `uconv` is not installed (tested on Ubuntu 20.04)  This may not be a bug persay, but we could add `icu-devtools` to list of libs for ubuntu in README?

https://github.com/smashedtoatoms/asdf-postgres/blob/b44003f60975bac1f7a2fe70fc6bf9fc79891268/bin/install#L47 

**Steps to reproduce**

`asdf install postgres 14.6` on linux installation without `unconv`

**Expected behavior**

Compilation without these errors:
```
/home/matt/.asdf/plugins/postgres/bin/install: line 45: uconv: command not found
/home/matt/.asdf/plugins/postgres/bin/install: line 49: [: : integer expression expected
```

**Screenshots**

n/a

**Additional context**

n/a

**Comment by smashedtoatoms** [comment]
Can you open a PR on the docs?

**Comment by smashedtoatoms** [comment]
This is fixed [in this commit](6ae18d71b06ae27ee14a7a8a3e59c02985b3bb43)
